### PR TITLE
update the network vlan on hardware detail

### PR DESCRIPTION
**Pull request by caberos**
[#issue1962](https://github.com/softlayer/softlayer-python/issues/1962)

image

### CODE REVIEW

**Comment by edsonarios** [comment]
Screenshots of the test result
image

File: SoftLayer/CLI/hardware/detail.py [hunk 1]
```diff
@@ -71,9 +72,11 @@ def cli(env, identifier, passwords, price, components):
     table.add_row(['last_transaction', last_transaction])
     table.add_row(['billing', 'Hourly' if result['hourlyBillingFlag'] else 'Monthly'])
 
-    vlan_table = formatting.Table(['type', 'number', 'id'])
+    vlan_table = formatting.Table(['type', 'number', 'id', 'Name', 'netmask'])
```
Comment (edsonarios) [inline SoftLayer/CLI/hardware/detail.py:75]: If the field Name is in capital letter, maybe the rest of the fields should the same format

File: SoftLayer/CLI/hardware/detail.py [hunk 2]
```diff
@@ -71,9 +72,11 @@ def cli(env, identifier, passwords, price, components):
     table.add_row(['last_transaction', last_transaction])
     table.add_row(['billing', 'Hourly' if result['hourlyBillingFlag'] else 'Monthly'])
 
-    vlan_table = formatting.Table(['type', 'number', 'id'])
+    vlan_table = formatting.Table(['type', 'number', 'id', 'Name', 'Netmask'])
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/hardware/detail.py:75]: Change the first letter of 'type', 'number' and 'id' to uppercase or the first letter of 'Name', 'Netmask' to lower case

**Comment by ramkishor-ch** [comment]
Hi,

I have "tested the changes".
Please find the screenshot for reference

Screen Shot: 1
Screenshot 2023-06-02 at 11 15 37 PM

Thanks,
Ramkishor Chaladi.

**Review by ramkishor-ch — APPROVED** [review]
Hi,
I have "tested the changes".
Approving the Pull Request.
Thanks,
Ramkishor Chaladi.

**Comment by JayasilanJibm** [comment]
Hi,

I have tested the changes. Looks fine to me.

Approving the pull request.

Thanks,
Jayasilan

Sharing the screenshot for further reference.

Screenshot 2023-06-12 at 12 20 19 PM

**Review by JayasilanJibm — APPROVED** [review]

**Pull request merged**